tests/behave/features/steps/legal_adviser.py

```diff
@@ -0,0 +1,28 @@
+from behave import given, when, then
+from features.helpers.fala import get_fala_url
+
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.common.by import By
+
+
+@given("I am a on the fala landing page")
+def step_fala_landing_page(context):
+    get_fala_url(context, "/")
+
+    WebDriverWait(context.driver, 10).until(EC.presence_of_element_located((By.ID, "content")))
+
+
+@when(u'I search for a legal adviser with postcode "{postcode}" and category "{category}"')
+def step_search_legal_adviser(context, postcode, category):
+    context.driver.find_element_by_xpath("//input[@type='text'][@name='postcode']").send_keys(postcode)
+    context.driver.find_element_by_xpath("//input[@name='categories'][@value='{}']".format(category))
```

Comment:
We don't do anything with the "category" element after finding it, is that intentional?

Reply:
Good catch. Turns out I wasn't even filtering by category 🤦‍♂